language/move-lang/src/parser/syntax.rs

```diff
@@ -627,48 +606,21 @@ fn maybe_parse_value(tokens: &mut Lexer) -> Result<Option<Value>, Error> {
             tokens.advance()?;
             Value_::Bool(false)
         }
-        Tok::U8Value => {
-            let mut s = tokens.content();
-            if s.ends_with("u8") {
-                s = &s[..s.len() - 2]
+        Tok::NumValue => {
+            // Num value should be parsed as the beginning of a address access
```

Comment:
That comment is confusing. Can you make it one of these?
(1) NumValue should NOT be parsed AT the beginning of AN address access
(2) If the number is followed by "::", parse it as the beginning of AN address access